src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -183,16 +184,29 @@ public async Task<OrganizationUserResetPasswordDetailsResponseModel> GetResetPas
     }
 
     [HttpPost("invite")]
-    public async Task Invite(string orgId, [FromBody] OrganizationUserInviteRequestModel model)
+    public async Task Invite(Guid orgId, [FromBody] OrganizationUserInviteRequestModel model)
```

Comment:
Seems like we're starting to get some more business logic in the controller. Do you have any plans/concern here?

Reply:
I agree, but it is authorization logic still - you only have to sort the groups and collections in this way because you're accessing it via this endpoint as a user. It doesn't apply to using the public api, for example. This is reflected by the fact that it requires `AuthorizationService` for this logic, which isn't available in the core layer. So I think it stays here for now, although I do agree that the endpoint itself is on the longer side.